tests/Api/Responses/Expected/shop/catalog_promotion/get_catalog_promotion_response.json

```diff
@@ -0,0 +1,4 @@
+{
+  "id": @integer@,
```

Comment:
There are no `@context` and related API Platform fields. The reason is that I wasn't able to include them in the shop context. In the admin context, everything works well

Reply:
But you should have the hydra date in response, I think that you have some problem with serialization.

Reply:
Adding json+ld header resolved the issue